CMakeLists.txt

```diff
@@ -86,6 +86,15 @@ endif()
 
 message(STATUS "Started CMake for ${PROJECT_NAME} v${PROJECT_VERSION}...\n")
 
+# option to use probe read/write checks
+if(DEFINED ENABLE_PROBE_READ_CHECK)
+  add_compile_definitions(ENABLE_PROBE_READ_CHECK=1)
```

Comment:
This definition is only needed for target `runtime`, you may use target_compile_definitions